cmd/examples/posix-oneshot/README.md

```diff
@@ -17,6 +17,10 @@ echo "foo" > /tmp/stuff/foo
 echo "bar" > /tmp/stuff/bar
 echo "baz" > /tmp/stuff/baz
 
+# Make a blank dir where we'll store the log data
+export LOG_DIR="/tmp/tlog"
```

Comment:
For a real usage of this tool, the log directory is really important. Burying the setting of it into the middle of the script doesn't do it justice.

I suggest either moving this to the first line of the script, or we can do something like the following to preserve any env variable the user explicitly sets:

```suggestion
LOG_DIR=${LOG_DIR:-/tmp/tlog}
```